src/components/line-tab/LineTab.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { SLineTab } from './styles/SLineTab';
+
+export interface ILineTabProps {
+  hover?: boolean;
+  activated?: boolean;
+  title?: string | undefined;
+  onClick: () => void;
+}
+
+export const LineTab: React.FC<ILineTabProps> = ({ ...props }) => {
+  return (
+    <SLineTab {...props}>
+      {props.title}
```

Comment:
There has to be some margin between title and the icon.
Current render.
<img width="91" alt="Screen Shot 2022-01-09 at 12 42 51 PM" src="https://user-images.githubusercontent.com/642960/148689585-9d9e2413-d69f-4fdc-937e-8bd95be4443b.png">

Figma:
https://www.figma.com/file/1hz5D4Q7pj5w0YrRw7hXbu/UI-Kit?node-id=13%3A1192

Reply:
These will be fixed on https://linear.app/caisy/issue/CY-659